nixos/modules/services/networking/unbound.nix

```diff
@@ -139,10 +262,24 @@ in
         RestartSec = "5s";
       };
     };
-
-    # If networkmanager is enabled, ask it to interface with unbound.
-    networking.networkmanager.dns = "unbound";
-
   };
 
+  imports = [
+    (mkRenamedOptionModule [ "services" "unbound" "interfaces" ] [ "services" "unbound" "settings" "server" "interface" ])
+    (mkChangedOptionModule [ "services" "unbound" "allowedAccess" ] [ "services" "unbound" "settings" "server" "access-control" ] (
```

Comment:
Any reason why those were removed? IIRC the RFC was meant as an addition to existing options (but correct me if I'm wrong).

Reply:
Indeed, but I thought it might make more sense to rename those to alert the user of changes in the module. For the `access-control` option, there are more options than just `allow` and `deny`, so the user should be able to use those too.

Reply:
Well, isn't the RFC meant as an addition for config options? Explicitly defined options for important stuff (and I'd consider the `interfaces` and `allowedAccess` sufficiently important) is still fine and with this we even have documentation and explicit type-checking for those as well.

> For the access-control option, there are more options than just allow and deny, so the user should be able to use those too.

How about fixing the option then?

Reply:
I disagree about modifying `allowedAccess` in case of future modifications upstream. I think renaming `interfaces` makes sense as the option doesn't change.